common/src/main/java/io/homeassistant/companion/android/common/data/websocket/impl/entities/ConversationResponse.kt

```diff
@@ -1,9 +1,10 @@
-package io.homeassistant.companion.android.common.data.websocket.impl.entities
-
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties
-
-@JsonIgnoreProperties(ignoreUnknown = true)
-data class ConversationResponse(
-    val response: ConversationSpeechResponse,
-    val conversationId: String?
-)
+package io.homeassistant.companion.android.common.data.websocket.impl.entities
```

Comment:
I think you're changing the line endings here, causing the entire file to show up as changed, instead of just the 2 changed lines. Can you undo that?

(There is another PR pending to set attributes to make this easier, as you've already discovered.)

Reply:
Indeed https://github.com/home-assistant/android/pull/5115/files#diff-1aa2d56c19899c60887fda68b38768529fd8ec55e0451a817dc3bbe7d49b3530